scripts/benchmark_submit.py

```
@@ -1,72 +1,394 @@
+    # Load all experiments
+    print(f"Loading {len(experiment_paths)} experiments...")
+    experiments: list[ExperimentMetadata] = []
+    for path in experiment_paths:
+        try:
+            exp = load_experiment_from_folder(path)
+            experiments.append(exp)
+            print(
+                f"  ✓ {exp.experiment_id}: {exp.results.success_rate:.1%} success, "
+                f"{exp.results.total_runs} runs",
+            )
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)